src/apps/ethereum/__init__.py

```diff
@@ -1,9 +1,12 @@
 from trezor import wire
 from trezor.messages import MessageType
 
+from apps.common import HARDENED
+
 
 def boot():
-    wire.add(MessageType.EthereumGetAddress, __name__, "get_address")
-    wire.add(MessageType.EthereumSignTx, __name__, "sign_tx")
-    wire.add(MessageType.EthereumSignMessage, __name__, "sign_message")
+    ns = [["secp256k1", HARDENED | 44, HARDENED | 60]]
```

Comment:
Could we write `44 | HARDENED` instead (I think it's more readable) or is that black's work?